pants

```diff
@@ -62,23 +61,24 @@ function tempdir {
 }
 
 function get_exe_path_or_die {
-  exe="$1"
+  local exe="$1"
```

Comment:
I didn't realize that function variables are global by default...I added `local` to lots of places, pardon the noise.